src/envinfo.js

```diff
@@ -1,16 +1,13 @@
 'use strict';
 
+require('@babel/polyfill');
```

Comment:
no need to add this if we use this config right https://babeljs.io/docs/en/babel-preset-env#usebuiltins

Reply:
I have tried it, but it not working on node 6.